backend/server/issue.go

```diff
@@ -336,6 +348,19 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 		if err != nil {
 			return err
 		}
+
+		if s.MetricReporter != nil {
+			s.MetricReporter.Report(&metric.Metric{
+				Name:  metricAPI.IssueUpdateMetricName,
+				Value: 1,
+				Labels: map[string]interface{}{
+					"type":  updatedComposedIssue.Type,
+					"key":   "status",
```

Comment:
wondering why here we use "key": "status", "value": updatedComposedIssue.Status

Instead of just "status": updatedComposedIssue.Status

Reply:
If so, the metric name should be named as "bb.issue.status-update", I think it's too specific. Besides, every metric name will map to a table name after we sync the data to pg.

Reply:
I thought it's actually better to use "bb.issue.status-update" because this maps to PATCH /issue/:issueID/status

Reply:
Also, the original requirement for calculating the activation metric is the first successful schema update. Thus a more accurate place is to add the metric at task scheduler https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/backend/runner/taskrun/scheduler.go?L289&subtree=true

Reply:
Updated, PTAL